### PR TITLE
docs: Updated the user logos on the readme

### PR DESCRIPTION
**Pull request by emreb**
#### Description

Changed the user logos to a single image in order to be dark mode friendly

#### Checklist 

- [x] The PR title has the correct prefix 
- [x] PR is linked to the corresponding issue
- [x] All commits are signed-off (`git commit -s ...`) to provide the [DCO](https://developercertificate.org/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,77 +45,12 @@ These APIs can be called via [cURL](#api-request), or in production via one of o
...
-  </tr>
-</table>
-
-_Using Cerbos? Open a PR to add your company._
+<p align="center">
+  <img src="https://cerbos.dev/assets/logos/readme_logos.png" alt="Cerbos"/>
+</p>
+
+
+_Using Cerbos? Please email us at devrel@cerbos.dev to get your logo added._
```
Comment (charithe) [inline README.md:53]:
```suggestion
_Using Cerbos? Let us know by emailing devrel@cerbos.dev._
```

**Review by charithe — COMMENTED** [review]
I think it would look better with some of the excess whitespace around the edges trimmed off.

**Comment by charithe** [comment]
This is how it looks with the dark theme. The border stands out more than the logos.

image

**Comment by emreb** [comment]
> This is how it looks like with the dark theme. The border stands out more than the logos.
> 
> image

I also updated the image at the origin, this is now how it looks
image

**Review by charithe — APPROVED** [review]
LGTM

**Pull request merged**